Documentation/archlinux_setup.md

```diff
@@ -0,0 +1,135 @@
+# 'container1.lxc' will then resolve on your host.
+#LXC_DOMAIN="lxc"
+EOF
+```
+
+### Modify the container template to use the bridge
+```bash
+cat << EOF > /etc/lxc/default.conf
+lxc.net.0.type = veth
+lxc.net.0.link = lxcbr0
+lxc.net.0.flags = up
+lxc.net.0.hwaddr = 00:16:3e:xx:xx:xx
+EOF
+```
+
+### Start the bridge
+```bash
+# Start the service
+systemctl start lxc-net
+
+# And then if you want to make sure it is enabled on boot as well
+systemctl enable lxc-net
+```
+
+### Clone the repository
+```bash
+git clone https://github.com/byteinternet/hypernode-vagrant
+```
+
+### Change directory to the new checkout
+```
+cd hypernode-vagrant
+```
+
+### Install the vagrant-lxc plugin
+```
+vagrant plugin install vagrant-lxc
+```
+
+### Make sure your SSH agent is running (optional)
+
+If you do not want to log in with `vagrant ssh` but want to SSH directly into the box instead you need to run an SSH agent with one or more keys loaded when you run `vagrant up`. The [provisioning script](https://github.com/ByteInternet/hypernode-vagrant/blob/master/vagrant/provisioning/hypernode.sh#L40) will copy your public key to the box for the `root` user and the non-privileged `app` user.
+
+```bash
+eval $(ssh-agent -s); ssh-add
+```
+
+The reason why you might want to SSH directly into the box instead of connecting with `vagrant ssh` is that if you use SSH forwarding (`ssh -A`, for more info [see this nice article by GitHub](https://developer.github.com/v3/guides/using-ssh-agent-forwarding/)) you will be able to connect to your production Hypernode if it also has the public key in your agent in its authorized keys file. This can be desirable for when you want to use the [hypernode-importer](https://support.hypernode.com/knowledgebase/migrating-your-magento-to-hypernode/#Option_2_Migrate_your_shop_via_Shell_using_hypernode-importer_8211_Magento_1_2) to copy the shop from your production Hypernode into the local Vagrant.
+
+### Initialize the environment
+```bash
+# So we do not accidentally use VirtualBox if that is also available on the system
+export VAGRANT_DEFAULT_PROVIDER=lxc
+vagrant up  # First run will install the 'vagrant-hypconfigmgmt' plugin
+vagrant up  # Second run will prompt for settings like PHP 5 or 7?
+vagrant up  # Will bring up the machine
+```
+
+For `lxc` make sure you have configured something other than `virtualbox` as the shared folder type. If you need two way sync I would suggest `nfs_guest` or `nfs`. 
+
+If you do not care and just want it to work, run this to set it to `rsync`. This will only make it sync from the host to the guest (copy everything to the container) and requires no setup.
+
+```bash
+# Change the stored file share setting from virtualbox to rsync
+sed 's/virtualbox/rsync/g' local.yml -i 
+```
+
+### Log in to the container
+
+If you did not have an SSH agent running you should run:
+```bash
+# Log in as the default vagrant user
+vagrant ssh
+
+# And then inside the container:
+
```

Comment:
Is there something missing here?

Reply:
no but I removed the newline now for clarity